tests/system/API/ResponseTraitTest.php

```diff
@@ -79,7 +79,7 @@ protected function makeController(array $userConfig = [], string $uri = 'http://
         }
 
         // Create the controller class finally.
-        $controller = new class ($this->request, $this->response, $this->formatter) {
+        return new class ($this->request, $this->response, $this->formatter) {
```

Comment:
I'm surprised Rector didn't get this one already! Must be related to the anonymous class.